layouts/partials/fragments/contact.html

```diff
@@ -33,71 +33,71 @@ <h2>{{- . | markdownify -}}</h2>
                       class="form-control"
                       type="text"
-                      data-validation="email"
-                      data-validation-error-msg="{{ .error | default (i18n "contact.defaultEmailError") }}"
-                      data-validation-error-msg-container="#email-error"
+                      contact-validation="email"
+                      contact-validation-error-msg="{{ .error | default (i18n "contact.defaultEmailError") }}"
+                      contact-validation-error-msg-container="#email-error"
                       placeholder="{{ with .text }}{{ . | markdownify }}{{ end }}"
                       required>
                     <div class="px-2 d-none" id="email-error"></div>
                   </div>
                 {{ end }}
-                {{ with $data.fields.phone }}
+                {{ with $contact.fields.phone }}
                   <div class="form-group">
                     <input
                       id="phone"
                       name="phone"
                       class="form-control"
                       type="text"
-                      data-validation="custom"
-                      data-validation-regexp="^([0-9,\+]+)$"
-                      data-validation-optional="true"
-                      data-validation-error-msg="{{ .error | default (i18n "contact.defaultPhoneError") }}"
-                      data-validation-error-msg-container="#phone-error"
+                      contact-validation="custom"
+                      contact-validation-regexp="^([0-9,\+]+)$"
+                      contact-validation-optional="true"
+                      contact-validation-error-msg="{{ .error | default (i18n "contact.defaultPhoneError") }}"
+                      contact-validation-error-msg-container="#phone-error"
                       placeholder="{{ with .text }}{{ . | markdownify }}{{ end }}">
                     <div class="px-2 d-none" id="phone-error"></div>
                   </div>
                 {{ end }}
               </div>
               <div class="col-md-6">
-                {{ with $data.fields.message }}
+                {{ with $contact.fields.message }}
                   <div class="form-group">
                     <textarea
                       id="message"
                       name="message"
                       class="form-control text-dark"
                       type="text"
-                      data-validation="length"
-                      data-validation-length="min1"
-                      data-validation-error-msg="{{ .error | default (i18n "contact.defaultButton") }}"
-                      data-validation-error-msg-container="#message-error"
+                      contact-validation="length"
+                      contact-validation-length="min1"
+                      contact-validation-error-msg="{{ .error | default (i18n "contact.defaultButton") }}"
+                      contact-validation-error-msg-container="#message-error"
```

Comment:
This change breaks the contact form. Search and replace did replace a bit too much here ;)

Reply:
Ha. seen will fix that :)